### ISSUE TITLE
SVG files with correct viewport dimensions?

### ISSUE THREAD

**Issue by laurent22** [ForkAwesome/Fork-Awesome#194]
I've noticed that all the SVG files have a viewport of 1536x1536. Obviously, it means that non-rectangular images like [this one](https://github.com/ForkAwesome/Fork-Awesome/blob/master/src/icons/svg/file-image-o.svg) are cropped at the bottom.

I'm wondering is there any build script I can use that would give me the svg with the correct viewport? Alternatively, shouldn't the source SVGs be set from the start with a viewport that corresponds to the content?

**Comment by tessus** [comment]
@xuv can you answer this? I know that we also use `width="1536" height="1792" viewBox="0 0 1536 1792"`.

I believe the sizing and the viewport has to do with the fact that icons go below and above the baseline. But @xuv can explain this in much better detail.

**Comment by xuv** [comment]
Yes. As @tessus notes it, the `viewport` information is used by the font build script to determine the position of the icons relative to the base line.

What we could do is use the `viewbox` information to display the icon correctly when viewed as SVG. It's something I started to test and have good results with. But I have not taken the time to fix this. If someone wants to try their hands at it. 

This has been already mentioned in issue #75

**Comment by tessus** [comment]
@xuv any update on this? unfortunately I don't know what and how to fix this.

**Comment by xuv** [comment]
@tessus Thx for the reminder. This is definitely on my todo list but will require quite some work. And so it sits on our tech debt list :)
